### DISCUSSION TITLE
Dead Project?

### DISCUSSION THREAD

**Discussion by etlweather** [mono/taglib-sharp#354] [General]
Just checking if there is a maintainer, or if someone else is maintaining this in a different project?

**Comment by decriptor** [comment]
Poorly maintained at best. There are some permission complications since I'm not longer at Microsoft. I'm curious to see what happens with the mono project being handed over to WineHQ.

I've thought about moving this library to a different organization so that I have some control over it.

**Comment by decriptor** [comment]
Having said that, there are two things I'd love happen. I really want to relicense this as MIT, but I haven't had much success figuring that out. Also, if there are others out there that would like to help maintain this with me that'd be amazing.
